mysql/tests/compose/mariadb.yaml

```diff
@@ -29,6 +30,7 @@ services:
       - MARIADB_REPLICATION_PASSWORD=my_repl_password
       - MARIADB_MASTER_HOST=mysql-master
       - MARIADB_MASTER_PORT_NUMBER=3306
+      - DB_REPLICATION_SLAVE_DUMP=false
```

Comment:
I'm fine with merging as it is, but in this case you could also add a comment to explain the reason of this